Encountify/Models/CustomPin.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.Maps;
+
+namespace Encountify.Models
+{
+    public class CustomPin : Pin
+    {
+        public string Name { get; set; }
+        public string Url { get; set; }
+        public Dictionary<string, object> Atts { get; set; }
```

Comment:
I couldn't think of any, but Atts might have a more informative name, so change if you can find it :D

Reply:
To be honest I just took this sample out of some code which had InfoWindows working and I'm not sure how this part works but it works there for I'm not sure what to change this to LOL
Maybe when ill try to simplify the CustomRenderer code, I'll find a more elegant way for everything since I think not every part of that code there is needed.